buildpipeline/DotNet-CoreFx-Trusted-Linux.json

```diff
@@ -73,17 +73,17 @@
     {
       "enabled": true,
       "continueOnError": false,
-      "alwaysRun": true,
-      "displayName": "Docker clean-up",
+      "alwaysRun": false,
```

Comment:
Good catch on the difference, but I think the intention was `true`, since "Initialize tools" is set to always run. If there's a machine state error that the cleanup will fix, I can see why this should be `true`, but I don't know one so I'm ok with this change.